### ISSUE TITLE
Decide Election Plan

### ISSUE THREAD

**Issue by mrocklin** [numfocus/elections#2]
I think that right now the plan is something like the following:

- [ ] Solicit nominations for board members
- [ ] Review nominations for board members
- [ ] Share results with current board, get their input for a final slate of nominees
- [ ] Publish Election with board nominees
- [ ] Market election, build up good participation among voters
- [ ] Review election results
- [ ] Share with board
- [ ] Publish results

A lot of the work here is also likely to be disseminating this information to the correct voting parties and annoying them sufficiently to ensure sufficiently high participation.

Am I missing anything here?

**Comment by mrocklin** [comment]
@lsilen @dutc checking in with you that a July 1st outcome of elections seems reasonable.  This feels like a long time to me, but it also seems about as fast as is possible given the mandated timings in the Board-provided election procedure.  (This timeline is actually pretty optimistic and assumes that nothing goes wrong.)

**Comment by lsilen** [comment]
I would love to see elections wrapped up sooner. What about -

- Solicit nominations for board members (April 2nd)
 - Close nominations and review (April 21, review 22-26)
 - Share results with current board, get their input for a final slate of nominees (April26-30)
 - Send out applications to nominees (May 1)
 - Application deadline (May 15th)
 - Election starts (May 20th)
 - Market election, build up good participation among voters
 - Close election and review results (June 10th) (three weeks)
 - Share with board - to be approved at 6/14 meeting
 - Publish results (June 17-19 )

**Comment by mrocklin** [comment]
Yeah, so to be clear, there are a few delays currently mandated in the board document

- One month nomination process
- Two week application process
- Nothing specified for voting
- General gaps for us to do some work and make some decisions ahead of time.

I'm totally happy to push back on those though (and it sounds like you are too, given your proposed schedule).  Let's see if we can get even more aggressive.

- Solicit nominations for board members (April 1st,  Monday)
 - Close nominations and review (April 14th, Sunday) (two weeks)
 - Share results with current board, get their input for a final slate of nominees (April 15-18 Monday-Thursday)
 - Send out applications to nominees (April 19th Friday)
 - Application deadline (April 27th Saturday) (one week ish)
 - Election starts (May 1st Wednesday) (four days to process applications
 - Market election, build up good participation among voters
 - Close election and review results (May 17th) (two-ish weeks)
 - Share with board
 - Publish results (May 22nd, May)

If we're ok changing mandates that schedule seems aggressive-but-doable to me.  

However, maybe you're mostly targetting the June 14th board meeting.  Is that correct?  Is the target here really just "get results to the board by June 14th? a couple weeks earlier doesn't really help us?"

**Comment by lsilen** [comment]
No problem adjusting the timeline outlined in the procedures doc. I wasn't targeting the 6/14 brd mtg, the dates just happened to align. There's also a meeting on 5/17 which works with this schedule.
Last time around, we received feedback that two weeks wasn't long enough for voting due to folks being out of town. I'd like to get the committee's feedback on this.

**Comment by mrocklin** [comment]
> due to folks being out of town

But like 90% of people will be in town, right?  I don't feel the need to ensure 100% or even 90% participation.  It may be that some people are sad that they weren't around for voting, but some sad people are probably inevitable with any path we might take.

**Comment by mrocklin** [comment]
(but in general I agree with making some space for committee members to weigh in)

**Comment by jaspajjr** [comment]
This may be a silly question, but why do people have to be in town to have a say? NumFocus is run remotely the majority of the time right? Why would this be any different?

**Comment by ivanov** [comment]
I think one thing worth clarifying here, is from a voter perspective, when will the ballot be finalized. Part of the charge of the election committee from the board is to "screen" or "vet" candidates prior to getting them applications. Would it make sense to then advertise the candidates as soon as they deemed eligible for the election, basically simultaneous to

> Send out applications to nominees (April 19th Friday)

Because otherwise if we wait until applications are due, then there's not really any time before the election starts for people to learn who the candidates are. I think announcing the candidates as soon as possible would also help with

> Last time around, we received feedback that two weeks wasn't long enough for voting due to folks being out of town.

Because then we make it close to a month from knowing who the candidates are until the last moment you can cast a vote for them. I can see how two weeks for the whole election can seem cramped, whereas if it is recast as two weeks to cast a vote, after two weeks of knowing who the candidates are, it's is less onerous on the voters.

**Comment by mrocklin** [comment]
OK, so the proposal here would be a rolling review process, where once someone comes in, we look at them, say "yup, looks like a reasonable candidate" and then add them to a list somewhere

Maybe we also give them an opportunity to immediately fill out an application so that that information goes wherever we list them?

This gives an advantage to people who are nominated early, but maybe that's ok.  We don't allow for any voting until we've closed nominations and collected applications properly.

**Comment by mrocklin** [comment]
Oh wait, you're not suggesting rolling releases of candidates, just proactive screening.  That seems fine with me.  It's a question of how well we can commit to screening people quickly.

**Comment by Cheukting** [comment]
What do we need before the nomination to be opened? Then we know when we can kick start the process.

**Comment by mrocklin** [comment]
Copying a comment from Slack

Reminder if people have time to take a look at the issues above (thanks those who have)
I think that we need to ...
- Converge on a proposed timeline (see [Election Plan and Timeline](https://github.com/numfocus/elections/issues/2))
- Write up some context and put it into something we can share with people (maybe just the README)  (see [Communicate Context to Community](https://github.com/numfocus/elections/issues/5))
- Start the process by soliciting nominations ([Solicit Nominees](https://github.com/numfocus/elections/issues/4))

The nomination process takes weeks, so starting it sooner rather than later would be good.  We can work on other pieces while that's going on.
If people can start this process by engaging on the election plan and timeline issue that would be good.  I think that we need two things:
- People in this committee to weigh in on both ...
    - Can we execute to this timeline?
    - Is this enough time for people to engage (there's a tension here between giving people time to engage and NumFOCUS being without an active board)
- Check in with the board about the proposed timeline (

**Comment by mrocklin** [comment]
@dutc you mentioned in the meeting that you preferred a longer nomination period because ...

1. We're not in a rush
2. We should be concerned about not getting enough nominations

My sense from the meeting is that @lsilen disagrees with point 1 (we need to move faster than a July date) and with point 2 (historically most nominations have come in the first week).  

Do you agree with these points?  Do you have other concerns about a two-week nomination cycle?

**Comment by mrocklin** [comment]
Here is an updated proposed schedule.  It maintains a 2-week nomination period.

- Solicit nominations for board members (April 8th, Monday)
- Close nominations and review (April 21st, Sunday) (two weeks)
- Share results with current board, get their input for a final slate of nominees (April 22-25th Monday-Thursday)
- Send out applications to nominees (April 26th Friday)
- Application deadline (May 4th Saturday) (one week ish)
- Election starts (May 8th Wednesday) (four days to process applications)
- Publicize election, build up good participation among voters
- Close election and review results (May 24th) (two-ish weeks)
- Share with board
- Publish results (May 29th, Wednesday)

I expect that we'll end up pushing back one of these phases, and actually land with a result on the first week of June.  

Any thoughts or objections to this plan?

**Comment by pavithraes** [comment]
We've had several delays since we started, here is the current tentative timeline we're following:

Election phases | Tentative dates
-- | --
Nomination period | June 7–June 28
Nominee application review (by election committee and NumFOCUS board) | June 28–July 12
Notify nominees and get confirmation | July 12–July 26
Voting period | July 26–Aug 9
Review election results (by NumFOCUS board) | Aug 9–Aug 23
Publish results announcing new board members | Late August / Early September

**Comment by tkoyama010** [comment]
@all-contributors please add @Cheukting for mentoring

**Comment by pavithraes** [comment]
Closing as complete. :)
